docs/api/regressor/peplot.rst

```diff
@@ -35,3 +35,4 @@ API Reference
     :members: PredictionError
     :undoc-members:
     :show-inheritance:
+    :noindex:
```

Comment:
This fixes the warning/error that `yellowbrick.regressor.residuals` was indexed twice.

Reply:
Thanks for making this and other similar fixes to make our documentation more robust.